docs/source/ops/index.rst

```diff
@@ -0,0 +1,65 @@
+.. Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+.. ops-ref_index:
+
+Ledger API Server pruning
+=========================
+
+DAML Ledger API servers may support pruning DAML contracts archived and transactions submitted
+before or at a given ledger offset, for example in order to reduce hot storage usage or to comply
```

Comment:
I think it's worth spending an extra sentence or two on the goals of pruning (reduction of hot storage requirements and removal of PII).

Reply:
Done.